### ISSUE TITLE
push to mute not working

### ISSUE THREAD

**Issue by KnightDiviN3r** [ottomated/CrewLink#222]
push to talk used to work when i first got crewlink but it stopped working suddenly. i have no idea what it is but only voice activity works. i've tried using other keybinds and stuff but i dont know

**Comment by TheSpringBoy** [comment]
Hi,
push to talk work only when you are in the amongus itself, not in CrewLink software.

**Comment by KnightDiviN3r** [comment]
> Hi,
> push to talk work only when you are in the amongus itself, not in CrewLink software.

yeah i have been in-game. it worked the first day i tried crewlink.

**Comment by meza** [comment]
Same here. I've tried everything in the book, push to _anything_ isn't working. Mute, Deafen, Talk - nothing.
I was in the game, not even just the lobby. Tried switching settings around, remapping keys, nothing.
